packages/core/src/keys.ts

```diff
@@ -23,6 +23,13 @@ export namespace CoreBindings {
    */
   export const SERVERS = 'servers';
 
+  // Binding Constant prefixes / tags
+  export const CONTROLLERS = 'controllers';
+  export const CONTROLLERS_TAG = 'controller';
+
+  // Key for Binding the BootStrapper Class
+  export const BOOTCOMPONENT = 'components.BootComponent';
```

Comment:
I am not very happy about the fact that this key is living inside core, I feel it creates too much coupling. Can we move this key to the boot component please?

Reply:
This key is used by `core` so I think the key should live there. The purpose is for a user to be able to bind a custom `BootStrapperComponent` to the key and get the expected behaviour.

Example:
`app.component(CustomBootStrapperComponent, CoreBindings.BOOTCOMPONENT)`